### PR TITLE
Convert the last two decorators to tracers and delete BaseDecorator

### PR DESCRIPTION
**Pull request by trask**

### CODE REVIEW

File: instrumentation/twilio-6.6/src/test/groovy/test/TwilioClientTest.groovy [hunk 1]
```diff
@@ -144,7 +145,7 @@ class TwilioClientTest extends AgentTestRunner {
         }
         span(1) {
           name "MessageCreator.create"
-          kind CLIENT
+          kind INTERNAL
```
Comment (anuraaga) [inline instrumentation/twilio-6.6/src/test/groovy/test/TwilioClientTest.groovy:148]: Is this change ok?

Comment (trask) [reply instrumentation/twilio-6.6/src/test/groovy/test/TwilioClientTest.groovy:148]: well... it seemed better than having nested `CLIENT` spans. do u think better to suppress the nested `apache-httpclient` spans?

Comment (anuraaga) [reply instrumentation/twilio-6.6/src/test/groovy/test/TwilioClientTest.groovy:148]: Ah yeah I think we'd normally suppress them.

Comment (trask) [reply instrumentation/twilio-6.6/src/test/groovy/test/TwilioClientTest.groovy:146]: done

**Review by anuraaga — APPROVED** [review]

**Pull request merged**